module/render.js

```diff
@@ -0,0 +1,73 @@
+import navListDisplay from './navigation-list-display.js';
+
+const bookName = document.getElementById('name');
+const author = document.getElementById('author');
+const output = document.querySelector('.list-elements');
+
+export default class Collection {
+  constructor() {
+    this.arr = [];
+  }
+
+  // class to check array items in the local storage
+
+  getBooks() {
+    if (localStorage.getItem('booksList') === null) {
+      this.arr = [];
+    } else {
+      this.arr = JSON.parse(localStorage.getItem('booksList'));
+    }
+  }
+
+  // class to update array in the local storage
+
+  UpdateLocalStorage() {
+    localStorage.setItem('booksList', JSON.stringify(this.arr));
+  }
+
+  pushListItem(item) {
```

Comment:
- [ ] **Required:** To improve the user experience of your app, publish the exiting books every time the [DOM is loaded](https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event). Also, [see](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) this resource for options and better understanding.